plugins/confluence.go

```diff
@@ -46,12 +48,26 @@ func (p *ConfluencePlugin) GetAuthorizationHeader() string {
 	return lib.CreateBasicAuthCredentials(p)
 }
 
+func validateURL(urlStr string) bool {
+	parsedURL, err := url.Parse(urlStr)
+	if err != nil {
+		return false
+	}
+
+	if parsedURL.Scheme != "https" {
+		return false
+	}
+
+	return true
```

Comment:
It is so a simple and unimportant function that I think we can shrink it to not take Nadlan for our eyes.
```suggestion
	if parsedURL, err := url.Parse(urlStr); err != nil || parsedURL.Scheme != "https" {
		return false
	}
	return true
```